distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering/Natural_Order.enso

```diff
@@ -26,55 +26,72 @@ compare text1 text2 =
     iter2 = BreakIterator.getCharacterInstance
     iter2.setText text2
 
-    is_digit=c -> c>=48 && c<=57
+    ## check if a single character is between '0' and '9'
+    ascii_code_zero = 48
+    ascii_code_nine = 57
+    is_digit=character -> character>=ascii_code_zero && character<=ascii_code_nine
 
+    ## Find the end of a number and then return the substring, value and new prev and next values
```

Comment:
What are `prev` and `next` values. I think explaining their meaning in the comment would be helpful.